presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,436 @@ Except for the limit on queued queries, when a resource group runs out of a reso
-  new queries will be rejected.
+2. Set the ``resource-groups.configuration-manager`` property to ``db``.
 
-* ``hardConcurrencyLimit`` (required): maximum number of running queries.
+3. Set up a relational database: The database should be accessible by Presto.
+   It will be used to store the configurations of your resource groups.
 
-* ``softMemoryLimit`` (required): maximum amount of distributed memory this
-  group may use before new queries become queued. May be specified as
-  an absolute value (i.e. ``1GB``) or as a percentage (i.e. ``10%``) of the cluster's memory.
+4. Create tables for resource groups and selection rules: You need to create tables in the
+   database that will store the definitions of your resource groups and the rules for selecting
+   the appropriate resource group for a given query.
 
-* ``softCpuLimit`` (optional): maximum amount of CPU time this
-  group may use in a period (see ``cpuQuotaPeriod``) before a penalty will be applied to
-  the maximum number of running queries. ``hardCpuLimit`` must also be specified.
+5. Specify the database in the Presto configuration: In the ``etc`` directory of your Presto installation,
+   create a file named ``resource-groups.properties``. In this file, add a line that specifies
+   the JDBC URL of your database: ``resource-groups.config-db-url = <jdbc_url>``, where ``<jdbc_url>`` is the
+   JDBC URL of your database.
+
+The database resource group manager loads the configuration from a relational database. With the
+Database Resource Group Manager, changes to the configuration in the database take effect immediately
+and do not require a restart of the Presto server. This allows for more flexibility and
+dynamic changes to the resource group configurations.
+
+Note : The supported databases are MySQL, PostgreSQL, and Oracle.
+
+.. code-block:: text
+
+    resource-groups.configuration-manager=db
+    resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups
+    resource-groups.config-db-user=username
+    resource-groups.config-db-password=password
+
+Here's an example of what the tables in your database might look like:
+
+Resource Groups Table
+---------------------
+
+``resource_group_id``: The ID of the resource group.
+
+``name``: The name of the resource group.
+
+``soft_memory_limit``: The maximum amount of memory that the resource group can use.
+
+``max_queued``: The maximum number of queued queries that the resource group can have.
+
+``hard_concurrency_limit``: The maximum number of queries that the resource group can run concurrently.
+
+Selectors Table
+---------------
+
+``resource_group_id``: The ID of the resource group that this selector applies to.
+
+``user_regex``: A regex pattern that matches the user names for this selector.
+
+``source_regex``: A regex pattern that matches the source names for this selector.
+
+Database Resource Group Manager Properties
+------------------------------------------
```

Comment:
nit: sub-section to `Database Resource Group Manager`. change to `^^^^^^^^^^^^^^^^^^^^^^^^^`